model/group.cc

```diff
@@ -408,7 +408,7 @@ namespace minsky
       (y-this->y())*sin(rotation()*M_PI/180);
     float dy=(x-this->x())*sin(rotation()*M_PI/180)+
       (y-this->y())*cos(rotation()*M_PI/180);      
-    float w=0.5*iconWidth*z,h=0.5*iconHeight*z;
+    float w=0.5*iWidth()*z,h=0.5*iHeight()*z;
```

Comment:
It's probably wise to do this change as a separate PR from the #1225 fix. That way, it can be accepted or rejected independently of the other changes.